Sming/SmingCore/Network/NetUtils.h

```diff
@@ -27,11 +27,12 @@ class NetUtils
 	static int pbufFindStr(pbuf *buf, const char* wtf, int startPos = 0);
 	static char* pbufAllocateStrCopy(pbuf *buf, int startPos, int length);
 	static String pbufStrCopy(pbuf *buf, int startPos, int length);
+	static char* strstrl(char* source, const char* sequence);
 
 	static bool FixNetworkRouting();
 
 	// Debug
-	void debugPrintTcpList();
+	static void debugPrintTcpList();
```

Comment:
Is this change related to Websocket Key comparison? If not - please move it to another PR.